pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -52,8 +54,22 @@ def get_created_at(self, obj):
 
     def get_updated_at(self, obj):
         """Get the timestamp of the highest version CollectionVersion's created timestamp."""
-        collection = self.context["highest_versions"][obj.pk]
-        return collection.pulp_created
+        added = Q(
```

Comment:
One of the new requirements from galaxy_ng for performance is to not perform queries here, but instead in the get_queryset method. This can be done with the `annotate` approach that is in use there. The issue with doing it here is that if you have N items in the list view it goes to the database N times, which they no longer want. I learned about this while doing the `deprecated` work.